src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java

```diff
@@ -1919,7 +1919,7 @@ AuthorizedAuthenticationState doAuthorization(final boolean requireLoginPermissi
             final AuthorizationIdentity authorizationIdentity = realmIdentity.getAuthorizationIdentity();
             final SecurityDomain domain = capturedIdentity.getSecurityDomain();
 
-            SecurityIdentity authorizedIdentity = Assert.assertNotNull(domain.transform(new SecurityIdentity(domain, authenticationPrincipal, realmInfo, authorizationIdentity, domain.getCategoryRoleMappers(), capturedIdentity.getPublicCredentials(), capturedIdentity.getPrivateCredentialsPrivate())));
+            SecurityIdentity authorizedIdentity = Assert.assertNotNull(domain.transform(new SecurityIdentity(domain, authenticationPrincipal, realmInfo, authorizationIdentity, domain.getCategoryRoleMappers(), IdentityCredentials.NONE, IdentityCredentials.NONE)));
             authorizedIdentity = authorizedIdentity.withPublicCredentials(publicCredentials).withPrivateCredentials(privateCredentials);
```

Comment:
Oops, you're right, on this line the correct credentials are added.  Sorry for the noise!